test_util/cluster_api.py

```diff
@@ -274,6 +274,7 @@ def get_test_app(self, custom_port=False):
             'cpus': 0.1,
             'mem': 32,
             'instances': 1,
+            'user': 'nobody',
```

Comment:
I'm thinking we need to grab the "user to use" out of an environment variable, than various integration test runners can set how they want / need